screens/home/HomeScreen.js

```diff
@@ -486,7 +486,14 @@ class HomeScreen extends PureComponent {
         if (galleryCount > 0) {
             // async loop
             for (const img of this.props.gallery) {
-                if (Object.keys(img.tags).length > 0) {
+                console.log(img.lat);
+                if (
+                    Object.keys(img.tags).length > 0 &&
```

Comment:
Do we need to check if `img.hasOwnProperty('tags')`? https://trello.com/c/W43YF722/87-undefined-is-not-an-object-evaluating-objectkeysotags

Reply:
Remove log

Reply:
We might also need to check string "null". It might be easier to check if the value is a float?